packages/core/src/Stores/ui-store.js

```diff
@@ -582,4 +585,14 @@ export default class UIStore extends BaseStore {
     setIsNativepickerVisible(is_nativepicker_visible) {
         this.is_nativepicker_visible = is_nativepicker_visible;
     }
+
+    @action.bound
+    toggleAccountTypesModal(is_visible) {
+        this.is_account_types_modal_visible = is_visible;
+    }
```

Comment:
I think we should use a default toggle and have optional setter if we use the word `toggle` in the method.
Alternatively, instead of `toggle` we can use `set` (e.g.: `setAccountTypesModal`) and just use the value passed in as the argument without a default toggler.

```suggestion
    toggleAccountTypesModal(is_visible = !this.is_account_types_modal_visible) {
        this.is_account_types_modal_visible = is_visible;
    }
```